src/ossl/aes.rs

```diff
@@ -2221,6 +2327,8 @@ impl MsgEncryption for AesOperation {
         Ok(outlen)
     }
 
+    /// Provides the expect buffer size for the provided input plaintext length
```

Comment:
I mentioned this in some similar function that it is not clear what buffer size it is. Also expect -> expected. Please check similar length functions in this file if it has matching clear wording:
```suggestion
    /// Provides the expected output ciphertext buffer size for the provided input plaintext length
```